### PR TITLE
chore: add fixed valueMappings for Autocomplete field

### PR DESCRIPTION
**Pull request by Jshhhh**
*Issue #, if available:*

*Description of changes:*
- add fixed valueMappings to Autocomplete field

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by scottyoung — APPROVED** [review]

File: packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap [hunk 1]
```diff
@@ -20715,6 +20725,51 @@ export default function MyFlexUpdateForm(props) {
           {...getOverrideProps(overrides, \\"Customtags\\")}
         ></TextField>
       </ArrayField>
+      <Autocomplete
+        label=\\"Label\\"
```
Comment (hein-j) [inline packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap:20729]: Would be great if you could add an integ test for this work. Or cut a ticket tracking that.
Would be a simple non-DS backed form.
Also, I don't think the label's gonna show up here. I guess speaking to your previous work, `labelHidden=false` should be conditionally added to Autocompletes that are not `shouldWrapInArrayField`?

Comment (Jshhhh) [reply packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap:20729]: Thanks
1.  Added cypress test for non array autocomplete field. 
2. Yeah, label wont show up in this PR. Ill rev the label PR with the conditional

File: packages/codegen-ui/lib/utils/form-component-metadata.ts [hunk 2]
```diff
@@ -94,6 +94,7 @@ export const mapFormMetadata = (form: StudioForm, formDefinition: FormDefinition
 
         if (
           (element.relationship ||
```
Comment (hein-j) [inline packages/codegen-ui/lib/utils/form-component-metadata.ts:96]: If typescript allows, can we just pare down this condition to 'valueMappings' in element?

File: packages/test-generator/integration-test-templates/src/FormTests/CustomDog.tsx [hunk 3]
```diff
@@ -44,6 +44,7 @@ export default function () {
         <Text>{`age: ${customFormCreateDogResults.age}`}</Text>
         <Text>{`email: ${customFormCreateDogResults.email}`}</Text>
         <Text>{`ip: ${customFormCreateDogResults.ip}`}</Text>
+        <Text>{`ip: ${customFormCreateDogResults.breed}`}</Text>
```
Comment (hein-j) [inline packages/test-generator/integration-test-templates/src/FormTests/CustomDog.tsx:47]: Nice

File: packages/codegen-ui-react/lib/forms/form-renderer-helper/all-props.ts [hunk 4]
```diff
@@ -92,7 +92,7 @@ export const addFormAttributes = (
     }
 
     // TODO: Allow for other relationship types once valueMappings available
```
Comment (hein-j) [inline packages/codegen-ui-react/lib/forms/form-renderer-helper/all-props.ts:94]: Would appreciate it if you can delete this comment

File: packages/codegen-ui/lib/utils/form-component-metadata.ts (hunk 2 above)
Comment (Jshhhh) [reply packages/codegen-ui/lib/utils/form-component-metadata.ts:96]: Yup. itll work

File: packages/codegen-ui-react/lib/forms/form-renderer-helper/all-props.ts (hunk 4 above)
Comment (Jshhhh) [reply packages/codegen-ui-react/lib/forms/form-renderer-helper/all-props.ts:94]: Got it. will delete in rev

**Review by hein-j — APPROVED** [review]

File: packages/test-generator/integration-test-templates/src/FormTests/CustomDog.tsx (hunk 3 above)
Comment (Jshhhh) [reply packages/test-generator/integration-test-templates/src/FormTests/CustomDog.tsx:47]: Thanks!

**Review by rtpascual — APPROVED** [review]

**Pull request merged**